### ISSUE TITLE
[Question] What is the optimizer of LLaVA?

### ISSUE THREAD

**Issue by 20191864218** [haotian-liu/LLaVA#1704]
### Question

Hello！I have a few questions:
1、What is the optimizer of LLaVA? Is it AdamW?
2、What is the Scheduler of LLaVA? Is Cosine Annealing with Linear Warmup?
3、If you do LROA fine-tuning, there are two parameters, --mm_projector_lr and --learning_rate, what do they mean?

Thank you!
